cabal-install/shell.nix

```diff
@@ -0,0 +1,46 @@
+{ nixpkgs ? import <nixpkgs> {}, compiler ? "default", doBenchmark ? false }:
```

Comment:
Are you committing this on purpose? I don't think this should go upstream?

Reply:
Same as above, a mistake. Will be reverted asap.